scripts/auto-rebase/rebase.sh

```diff
@@ -781,6 +780,47 @@ update_images() {
             "${REPOROOT}/packaging/crio.conf.d/microshift_${goarch}.conf"
     done
 
+    title "Rebasing operator-lifecycle-manager manifests"
+    for goarch in amd64 arm64; do
+        # Get the images for the OLM operator from ${STAGING_DIR}/release_${arch}.json:
+        # - operator-lifecycle-manager
+        # - operator-registry
+        # - kube-rbac-proxy
+        arch=${GOARCH_TO_UNAME_MAP["${goarch}"]:-noarch}
+
+        cat <<EOF > "${REPOROOT}/assets/optional/operator-lifecycle-manager/release-${arch}.json"
```

Comment:
Do we need this file at all? I thought we're changing images directly in the `.$arch.yaml` files. Also this PR contains these files without images filled in

Reply:
I can remove the template in the shell script, as there is a file in assets already. That file is empty, so the `rebase.sh images` Will update with the proper image versions.